src/main/java/com/verily/genomewarp/utils/GvcfToVcfAndBed.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2016 Verily Life Sciences LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.verily.genomewarp.utils;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Helper class for extracting a canonical VCF and a BED file from a gVCF file
+ */
+public class GvcfToVcfAndBed {
+
+  private static final Logger logger = Logger.getLogger(VcfToVariant.class.getName());
+
+  /**
+   * Solve Issue #2 - Extract variant VCF and BED from (tentative) gVCF file
+   * 
+   * @param rawQueryVcf user-provided (g?)VCF path
+   * @param vcfFromGvcf intermediate canonical VCF path
+   * @param bedFromGvcf intermediate BED path
+   * @return true if query VCF is a gVCF, false otherwise
+   */
+  public static boolean saveVcfAndBedFromGvcf(String rawQueryVcf, String vcfFromGvcf,
+      String bedFromGvcf) throws IOException {
+    PrintWriter outBed = new PrintWriter(Files.newBufferedWriter(Paths.get(bedFromGvcf), UTF_8));
```

Comment:
Please use a try-with-resources (https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) that includes all three of these resources to ensure all files get closed properly.